static/locales/en-US.yaml

```diff
@@ -93,6 +93,10 @@ Subscriptions:
   'Getting Subscriptions. Please wait.': Getting Subscriptions. Please wait.
   Refresh Subscriptions: Refresh Subscriptions
   Load More Videos: Load More Videos
+  Tabs:
```

Comment:
It might make sense to reuse the existing strings from the channel page, as they have the same meaning. That way we won't have to wait for everyone to translate them before the RC, also stops us from having too many duplicate strings for the same thing.

Reply:
It's awkward either way we do this, as the strings are technically located in a place that's meant for the Channel view only. The right way to do it would be to move the strings outside of the Channel / Subscription objects and move it to a Neutral / Global location for reusable strings.

Reply:
I like the idea of moving them to a global location, as we have a bunch of other strings that are duplicated, so having a global location we could move stuff to would be good.